qutebrowser/mainwindow/prompt.py

```diff
@@ -630,6 +630,18 @@ def __init__(self, question, parent=None):
 
         self._to_complete = ''
 
+    def _directories_hide_show_model(self, path):
+        """Get rid of non-matching directories."""
+        try:
+            num_rows = self._file_model.rowCount(self._root_index)
+            for row in range(num_rows):
+                index = self._file_model.index(row, 0, self._file_model.index(path))
```

Comment:
I don't really understand the `self._file_model.index(path)` as parent here - you use `self._root_index` for `rowCount` above, so wouldn't the parent here need to be `self._root_index` as well?

Reply:
This is the product of 1 am coding. To answer your question, I'm pretty sure the indexes are exactly the same, but of course one is better than the other.

Reply:
Changed it to `self._root_index`.